provided.al2/graalvm/java11/cookiecutter-aws-sam-graalvm-gradle/cookiecutter.json

```diff
@@ -0,0 +1,7 @@
+{
+    "project_name": "Name of the project",
+    "runtime": "java11",
```

Comment:
Since this will actually use `provided.al2` as runtime, we don't need to mention this here.

Reply:
So remove it or change to `"runtime": "provided.al2"`?

Reply:
Looking at this file, I think we can just remove it https://github.com/aws/aws-sam-cli-app-templates/blob/9b488d5c4b2b3c3bd34bed871bfcd5ab4a00abc4/provided.al2/rust/cookiecutter-aws-sam-hello-rust/cookiecutter.json